### PR TITLE
Add confichat to README.md

### PR DESCRIPTION
**Pull request by 1runeberg**
- Added ConfiChat to Community Integration > Web & Desktop
- Added ConfiChat to Community Integration > Mobile

### CODE REVIEW

**Comment by jmorganca** [comment]
@1runeberg possible to rebase?

**Comment by 1runeberg** [comment]
done :)

File: README.md [hunk 1]
```diff
@@ -309,7 +309,8 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [PyOllaMx](https://github.com/kspviswa/pyOllaMx) - macOS application capable of chatting with both Ollama and Apple MLX models.
 - [Claude Dev](https://github.com/saoudrizwan/claude-dev) - VSCode extension for multi-file/whole-repo coding
 - [Cherry Studio](https://github.com/kangfenmao/cherry-studio) (Desktop client with Ollama support)
-  
+- [ConfiChat](https://github.com/1runeberg/confichat) (Lightweight, standalone, multi-platform, and privacy focused LLM chat interface with optional encryption)
```
Comment (jmorganca) [inline README.md:312]:
```suggestion
```

Possible to add it to one section for now? I think the Terminal section might be best?

Comment (1runeberg) [reply README.md:312]: It's not a terminal client though. It's a standalone gui. It has both Mobile and Desktop versions. 

Is there an issue having the app on both sections (Desktop & Mobile)? I have pre-compiled builds for all platforms in the repo if you need to verify :)

Comment (jmorganca) [reply README.md:312]: Ah, sorry I misread the diff! This is okay for now

**Review by jmorganca — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (1runeberg) [reply README.md:312]: Thanks Jeffrey! 🎉

**Pull request merged**